Makefile

```diff
@@ -74,10 +74,10 @@ deployoperator:
 	cd deploy && ./deploy.sh
 
 runmain: build-postgres-operator
-	postgres-operator --kubeconfig=/etc/kubernetes/admin.conf
+	KUBECONFIG=/etc/kubernetes/admin.conf postgres-operator
 
 runapiserver: build-pgo-apiserver
-	apiserver --kubeconfig=/etc/kubernetes/admin.conf
+	KUBECONFIG=/etc/kubernetes/admin.conf apiserver
```

Comment:
Hmm...why not put this uptop as `KUBECONFIG ?= /etc/kubernetes/admin.conf`?

Reply:
I removed these in an amended commit. It seemed like they weren't being used.